css-squid/src/main/java/org/sonar/css/CssAstScanner.java

```diff
@@ -106,6 +106,18 @@ public static SourceFile scanSingleFile(File file, SquidAstVisitor<LexerlessGram
       .subscribeTo(CssGrammar.AT_RULE)
       .build());
 
+    if (conf.getComputeComplexity()) {
+      builder.withSquidAstVisitor(CounterVisitor.<LexerlessGrammar>builder()
+        .setMetricDef(CssMetric.COMPLEXITY)
+        .subscribeTo(CssGrammar.CLASS_SELECTOR, CssGrammar.ATTRIBUTE_SELECTOR, CssGrammar.TYPE_SELECTOR, CssGrammar.ID_SELECTOR, CssGrammar.PSEUDO, CssGrammar.AT_RULE)
+        .build());
+    }
+
+    builder.withSquidAstVisitor(CounterVisitor.<LexerlessGrammar>builder()
```

Comment:
Fixed in https://github.com/racodond/sonar-css/commit/d64ecb6776c780fda003889c0d63ec8bed6d77e8